<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureData</class>
 <widget class="QDialog" name="FeatureData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>feature data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_dialogName">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;observations, readings and statistic of the selected feature&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget_selectedFeature">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="observations">
      <attribute name="title">
       <string>observations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tableView_observation"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="readings">
      <attribute name="title">
       <string>readings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,8">
         <item>
          <widget class="QLabel" name="label_readings">
           <property name="text">
            <string>used reading types</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_readings"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_readings"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_statistic">
      <attribute name="title">
       <string>statistic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_displayFunction">
           <property name="text">
            <string>displayed function statistic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_displayedFunction"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_displayedfunctionStatistic"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_covarianceMatrices">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_sigmaXXaposteriori">
          <attribute name="title">
           <string>Qxx 치 posteriori</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,1">
             <item>
              <widget class="QLabel" name="label_s0aposteriori">
               <property name="text">
                <string>s0 치 posteriori:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_s0aposterioriValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView_qxxAposteriori"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_sigmaXXapriori">
          <attribute name="title">
           <string>Sxx 치 priori</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QLabel" name="label_s0apriori">
               <property name="text">
                <string>s0 치 priori</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_s0aprioriValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView_sxxApriori"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_protocol">
      <attribute name="title">
       <string>protocol</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTextBrowser" name="textBrowser_protocol"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
